operator/main.py

```diff
@@ -211,10 +212,13 @@ def validate_volume_request(obj):
                       " specified")
         return False
 
+    # should'nt this come under disperse volume?? Not replica??
     if subvol_bricks_count > 1:
+        # Why not range(0, 1): since / is always == 1 ??
```

Comment:
More than 1 when the Storage is "Distributed Replicate" or "Distributed Disperse"

Reply:
Thanks